localstripe/resources.py

```diff
@@ -121,7 +120,10 @@ def __init__(self, id=None):
 
             self.livemode = False
 
-            key = self.object + ':' + self.id
+            if self.id:
+                key = self.object + ':' + self.id
+            else:
+                key = self.object
```

Comment:
This change and the one above (that I made you do sorry about that) are not needed anymore with the last commit version, no?